app/controllers/queue_items_controller.rb

```diff
@@ -1,8 +1,9 @@
 class QueueItemsController < ApplicationController
   before_action :require_login
+  before_action :set_items, only: [:index, :update_queue]
+  helper_method :list_orders
```

Comment:
@tomtomecek This is old code from my previous iteration of the update_queue method.  Sorry, I forgot to remove it.